src/main/java/mantle/blocks/abstracts/MultiItemBlock.java

```diff
@@ -15,21 +15,21 @@
 public class MultiItemBlock extends ItemBlock {
 
     private String blockType[];
-    private String unlocalizedName;
+    private String unlocalizedNameSecond;
```

Comment:
rather than renaming it... pretty sure you can add a Gson annotation to change the serialized name

Reply:
think i did it correct please check